conf/messages-ecc.en

```diff
@@ -86,4 +88,8 @@ cds.subscription.information.bullet2=National Insurance number (if you are an in
 cds.subscription.information.bullet3=your Unique Taxpayer Reference (UTR). You can {0}.
 cds.subscription.information.bullet3.link=find a lost UTR number (opens in a new window or tab)
 
-cds.navigation.subscribe=Subscribe
\ No newline at end of file
+cds.navigation.subscribe=Subscribe
+
+subscription.check-your-email.no=No, I need to change this email address
+
+cds.form.contact-details=Your address
```

Comment:
So what about company address?

Previously we had Address and now we should have Your address or Company address. You didn't defined the Company address.

Additionally just to confirm, because as I remember "address" part started with capital letter in the prototype. Is that incorrect there?

Reply:
This is the original definition
`cds.form.contact-details=Address`
The other types of entities use these labels
```
cds.form.partner-address=Partnership address
cds.form.company-address=Company address
cds.form.business-details=Organisation address
```

Reply:
Hmm, just wondering, because when I checked the Company journey at the end we're displaying just Address